### ISSUE TITLE
Login for non superusers

### ISSUE THREAD

**Issue by gio82** [geex-arts/django-jet#147]
If I try to login with a non superuser I get this error:

```
KeyError at /admin/
u'admin_url'
Request Method:	GET
Request URL:	http://localhost:8000/admin/
Django Version:	1.10.3
Exception Type:	KeyError
Exception Value:	
u'admin_url'
```

```
/jet/templatetags/jet_tags.py in jet_get_menu
    all_aps = []
    apps = []
    pinned_apps = []
    for app in app_list:
        if not current_found:
            for model in app['models']:
                if context['request'].path.startswith(model['admin_url']): ...
                    model['current'] = True
                    current_found = True
                    break
            if not current_found and context['request'].path.startswith(app['app_url']):
                app['current'] = True
```

**Comment by gio82** [comment]
This is an hot fix I'm using:

```
                try:
                    if context['request'].path.startswith(model['admin_url']):
                        model['current'] = True
                        current_found = True
                        break
                except: pass
```

**Comment by f1nality** [comment]
@gio82 Hello!
This was fixed in this PR https://github.com/geex-arts/django-jet/pull/144
Will be released in 1.0.3
